MonacoEditorTestApp/MainPage.xaml.cs

```diff
@@ -189,22 +168,22 @@ private void ButtonHighlightRange_Click(object sender, RoutedEventArgs e)
             this.Editor.Decorations.Add(
                 new IModelDeltaDecoration(new Range(3, 1, 3, 10), new IModelDecorationOptions()
                 {
-                    ClassName = new CssLineStyle() // TODO: Save these styles so we don't keep regenerating them and adding new ones.
+                    ClassName = new CssLineStyle(Editor) // TODO: Save these styles so we don't keep regenerating them and adding new ones.
                     {
                         BackgroundColor = new SolidColorBrush(Colors.DarkRed),
-                    },
-                    InlineClassName = new CssInlineStyle()
+                    }.Name,
+                    InlineClassName = new CssInlineStyle(Editor)
                     {
                         ForegroundColor = new SolidColorBrush(Colors.White),
                         FontWeight = FontWeights.Bold,
-                        FontStyle = Windows.UI.Text.FontStyle.Italic
-                    },
+                        FontStyle = FontStyle.Italic
+                    }.Name,
                     HoverMessage = new string[]
                     {
                         "This is a test message.",
                         "*YES*, **it is**."
                     }.ToMarkdownString(),
-                    Stickiness = TrackedRangeStickiness.NeverGrowsWhenTypingAtEdges
+                    Stickiness = (int)TrackedRangeStickiness.NeverGrowsWhenTypingAtEdges
```

Comment:
Shouldn't the Stickiness property be typed to the Enum still?

Reply:
Fixed via https://github.com/hawkerm/monaco-editor-uwp/pull/30/commits/0c43cc2223eff74e4f1c1f7ad4f0ce77372a6560